### DISCUSSION TITLE
removing pools and merging plots in the same pool

### DISCUSSION THREAD

**Discussion by shahinshahim** [Chia-Network/chia-blockchain#8527] [Ideas]
1. self pools with no plots which were made accidently by the user cannot be deleted.
2. I have pools with just 1 or 3 plots although I have a pool in which most of my plots are being farmed. although they're all in the same pool but considering the pool minimum amount  of 0.01 XCH to payout; these multiple little pools with like one plot are kind of a problem. 
3. I think it would be great to have a feature for deleting the pools we do not want to have them or to migrate and merge the whole plots completely in just one pool.  
Thank you.

![1](https://user-images.githubusercontent.com/90974015/133892697-d56f9aae-5bc1-4854-a58e-9ac97e2b4767.jpg)
![2](https://user-images.githubusercontent.com/90974015/133892657-b49702cd-7e7e-4599-b492-798eb135529f.jpg)
![3](https://user-images.githubusercontent.com/90974015/133892658-b2770731-2ff8-4e88-8674-dcfa67adacf9.jpg)

**Comment by loppefaaret** [comment]
it is impossible to merge plots from one plotNFT, into another - the entire contents of the plot is based on a key from the plotNFT... 
i suggest abandon the plotNFT with 3 plots, and replot those 3 plots under the correct plotNFT
perhaps wait for a pool payout, and then do it

**Comment by BrandtH22** [comment]
@shahinshahim , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
